mindsdb/integrations/handlers/lindorm_handler/requirements.txt

```diff
@@ -0,0 +1,3 @@
+pyphoenix
+phoenixdb
+protobuf==3.20.*
```

Comment:
Is protobuf required here?

Reply:
Hi @ZoranPandovski it was using same requirements for the phoenixdb handler. It was there present in that handler's requirements (for phoenixdb) that's why I've put it there .